### ISSUE TITLE
[audible][0.3.0] java.lang.NullPointerException in com.softenido.audible

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jareens/softenido#31]
```
ofuscated:

java.lang.RuntimeException: Unable to start activity 
ComponentInfo{com.softenido.audible/com.softenido.audible.AudibleMain}: 
java.lang.NullPointerException
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1956)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:1981)
at android.app.ActivityThread.access$600(ActivityThread.java:123)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1147)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:137)
at android.app.ActivityThread.main(ActivityThread.java:4464)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:511)
at 
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:822)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:589)
at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.NullPointerException
at com.softenido.audible.AudibleMain.l(Unknown Source)
at com.softenido.audible.AudibleMain.onCreate(Unknown Source)
at android.app.Activity.performCreate(Activity.java:4519)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1049)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1920)
... 11 more

unofuscated:

java.lang.RuntimeException: Unable to start activity 
ComponentInfo{com.softenido.audible/com.softenido.audible.AudibleMain}: 
java.lang.NullPointerException
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1956)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:1981)
at android.app.ActivityThread.access$600(ActivityThread.java:123)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1147)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:137)
at android.app.ActivityThread.main(ActivityThread.java:4464)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:511)
at 
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:822)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:589)
at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.NullPointerException
at com.softenido.audible.AudibleMain.void loadHeadAndBody()(Unknown Source)
at com.softenido.audible.AudibleMain.void onCreate(android.os.Bundle)(Unknown 
Source)
at android.app.Activity.performCreate(Activity.java:4519)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1049)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1920)
... 11 more



```

Original issue reported on code.google.com by `flik...@gmail.com` on 30 Nov 2012 at 12:16

**Comment by GoogleCodeExporter** [comment]
```
fixed at 53cbdc2baf5e0ff0d2cdd3fca622c5c37ca883b1
```

Original comment by `flik...@gmail.com` on 30 Nov 2012 at 12:27
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 53cbdc2baf5e.
```

Original comment by `flik...@gmail.com` on 11 Mar 2013 at 7:06
